2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,7 +18,14 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
```

Comment:
This looks good 👍 
Small point: Keep an eye on the spacing / indentatino of your code as this can make it easier for other developers to read it.